src/main/java/filter_post/interface_adapters/FilterPostController.java

```diff
@@ -0,0 +1,26 @@
+package filter_post.interface_adapters;
+
+import filter_post.use_case.FilterPostInputBoundary;
+import filter_post.use_case.FilterPostRequestModel;
+
+public class FilterPostController {
+    private final FilterPostInputBoundary interactor;
+
+    /**
+     * Initialize a FilterPostController object.
+     * @param interactor    A FilterPostInputBoundary object that specifies the functionality of the use case.
```

Comment:
If I were to be knit picky I would change the documentation to be something like "A FIlterPostInputBoundary object that runs the use case." "~ object that specifies the functionality of the uc" is a little confusing and if you want to mention that the specific functionality is described in that interface's documentation you could add a line that says exactly that. ex: "The functionality of the FilterPostInputBoundary is in its own documentation."